packages/common/src/ceramic-api.ts

```diff
@@ -75,7 +74,8 @@ interface CeramicCommon {
  * available users can provide any object containing an authenticated DID instance.
  */
 export interface CeramicSigner extends CeramicCommon {
-  did: DID | undefined
+  did: ThreadedDid | undefined
+  didVerifier: DidVerifier | undefined
```

Comment:
I'm guessing a `DidVerifier` is used to verify did signatures?  The http-client doesn't need to ever verify DID signatures, it only needs to create them, so this probably shouldn't be part of the common interface that is shared between the core and http client implementations

Reply:
I need a better name for that. It's responsible for managing the threaded did instances, so that data can flow properly. 

It is needed, since if someone does a late authentication for a did, the caching needs to be updated.

Reply:
It's needed in the http-client?  Why?  It doesn't use it now and seems to be fine, we don't need to use threads in the client.